pkg/gofr/crud_handlers.go

```diff
@@ -34,6 +34,14 @@ type Delete interface {
 	Delete(c *Context) (interface{}, error)
 }
 
+type TableName interface {
+	TableName() string
+}
+
+type RestPath interface {
+	RestPath() string
+}
+
```

Comment:
can we update the names of the interface it is not right, you can refer go docs

https://go.dev/doc/effective_go#interface-names